gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,7 +1,7 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.6-bin.zip
-distributionSha256Sum=7ba68c54029790ab444b39d7e293d3236b2632631fb5f2e012bb28b4ff669e4b
+distributionSha256Sum=1b6b558be93f29438d3df94b7dfee02e794b94d9aca4611a92cdb79b6b88e909
```

Comment:
We need to revert this Gradle version bump.  We are blocked on upgrading Gradle until all our other repos can also be upgraded.  This restriction is to support Gradle included builds between SDK, app, and bip-39, so we bump Gradle, Kotlin, and Android Gradle Plugin in lockstep between these projects to avoid compatibility issues.  All of this is currently blocked by incompatibility in the Rust Gradle Plugin with AGP 7.4.1+ in the SDK.

Reply:
reverted to 7.6